website/server/models/group.js

```diff
@@ -376,6 +376,7 @@ schema.methods.removeGroupInvitations = async function removeGroupInvitations ()
   let userUpdates = usersToRemoveInvitationsFrom.map(user => {
     if (group.type === 'party') {
       user.invitations.party = {};
+      user.invitations.parties = [];
```

Comment:
Yes, I'll fix it... And I noticed that there are still a lot of references to ``invitations.party`` in the code, so I need to update them all...